charts/sonarqube/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [5.4.1]
+* Refactor templating of ConfigMap for sonar.properties
```

Comment:
I would be a little bit more explicit here, saying that we also fix the setting of sonarSecretKey without being strictly dependent on the sonar.properties parameter.